deploy/application/php/php-apps.md

```diff
@@ -16,6 +16,28 @@ PHP is available on our platform with the branches 5.6, 7.2, 7.3, 7.4 and 8.0. Y
 
 The HTTP server is [Apache 2](https://httpd.apache.org/), and the PHP code is executed by [PHP-FPM](https://php-fpm.org/).
 
+## Memory Limit
+
+A smaller part of your application's memory is defined as the memory limit to allow workers to run efficiently, here is the memory limit for each flavor:
```

Comment:
I'm not sure to really understand what's this limit is when I read this sentence. I feel like this says `we only use 2048M of memory even though you have 32GB in total`. But that may be only me.

IIRC, this is the memory limit of each worker spawned by the `php-fpm` process.